### ISSUE TITLE
The inconsistency of Wav2VecFeatureExtractor in unsupervised wav2vec

### ISSUE THREAD

**Issue by andybi7676** [facebookresearch/fairseq#4464]
## 🐛 Bug

When I am running some experiments about unsupervised wav2vec, I found there are two versions of the class `Wav2VecFeatureExtractor`.  One is in `fairseq\examples\wav2vec\unsupervised\scripts\wav2vec_extract_features.py`, while the other is in `fairseq\examples\wav2vec\unsupervised\scripts\wav2vec_cluster_faiss.py`.  They are pretty the same, except the one in `wav2vec_cluster_faiss.py` does not apply layer_norm on the loaded waveform. And according to the original paper of wav2vec2.0, I think the implementation in `wav2vec_extract_features.py` is more suitable as it follows the config of the pretrained model to decide whether to apply normalization to the loaded waveform or not. Thus, I suggest removing the `Wav2VecFeatureExtractor` in `wav2vec_cluster_faiss.py` and importing the `Wav2VecFeatureExtractor` from `wav2vec_extract_features.py`. Also, `wav2vec_apply_cluster_faiss.py` should also be modified to import the `Wav2VecFeatureExtactor` from `wav2vec_extract_features.py` instead of `wav2vec_cluster_faiss.py`. 

### To Reproduce

Steps to reproduce the behavior (**always include the command you ran**):

1. Checkout the class `Wav2VecFeatureExtractor` in `fairseq\examples\wav2vec\unsupervised\scripts\wav2vec_extract_features.py`, and  `fairseq\examples\wav2vec\unsupervised\scripts\wav2vec_cluster_faiss.py`.

### Environment

 - fairseq Version (e.g., 1.0 or main):
 - PyTorch Version (e.g., 1.0)
 - OS (e.g., Linux):
 - How you installed fairseq (`pip`, source):
 - Build command you used (if compiling from source):
 - Python version:
 - CUDA/cuDNN version:
 - GPU models and configuration:
 - Any other relevant information:
